test/common/fit_common.py

```diff
@@ -867,7 +869,7 @@ def node_select():
             if str(fitargs()['sku']) in json.dumps(skuentry):
                 skuid = skuentry['id']
         # Collect node IDs
-        catalog = rackhdapi('/api/2.0/nodes')
+        catalog = rackhdapi('/api/2.0/nodes?type=compute')
         if skumap['status'] != 200:
```

Comment:
Yes, skumap['status'] is an error. Looks like that's been there since the beginning. Will revise.

catalog['json'] is called several times, but 'catalog' variable is the result from one API call. We could create yet another variable to hold catalog['json'] , but this is not really gaining us anything. I say leave it.